package.json

```diff
@@ -25,15 +25,15 @@
     "publish:dry": "fdz-deploy --dry",
     "publish:major": "fdz-deploy --major",
     "publish:minor": "fdz-deploy --minor",
-    "publish:patch": "fdz-deploy --patch"
+    "publish:patch": "fdz-deploy --patch",
+    "coveralls": "npm test && ls -la ./coverage && cat ./coverage/lcov.info | coveralls"
```

Comment:
Can't we remove the `npm test` part? That is already done before

Reply:
Well caught 😎